lib/tictactoe/game.rb

```diff
@@ -2,38 +2,39 @@
 require "./lib/tictactoe/validation"
 
 module Tictactoe
-  class PositionOutOfRange < StandardError; end
 
   class Game
     attr_reader :board
 
-    def initialize(validation, stdin: $stdin)
+    def initialize(validation, messages, io)
       @validation = validation
-      @stdin = stdin
-      puts "Welcome to Tic-Tac-Toe"
-      puts "Player one uses 'X' and player two uses 'O'"
+      @messages = messages
+      @io = io
+      @io.output(@messages.get_message("welcome"))
+      @io.output(@messages.get_message("symbols"))
       @symbols = []
       @board = Board.new
+      @game = true
```

Comment:
`@game` is also the name of this class, so I could misread this variable as an instance of Game class. Can you think of a better name to capture the spirit of this variable?